recordings/auth-api_3443157535/changes-user-s-password_3225638244/recording.har

```diff
@@ -47,10 +47,10 @@
             },
             {
               "name": "host",
-              "value": "master.staging.saleor.cloud"
+              "value": "jwt-expire-test.api.saleor.rocks"
```

Comment:
Shouldn't we omit hostname and other instance-specific headers?
Or maybe whitelist only those meaningful?

Reply:
These are not used for matching, and will not be regenerated when the test remains unchanged. I'm neutral on this change, I don't see this information as something that needs obfuscation.

Reply:
I know they are not used, but it generates bigger diffs. Out of scope of this PR though.